### PR TITLE
feat: test env setup

### PR DESCRIPTION
**Pull request by pi0trpietruszewski**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,13 +39,26 @@
...
+  "jest": {
+    "preset": "react-native",
+    "modulePathIgnorePatterns": [
+      "<rootDir>/examples/",
+      "<rootDir>/lib/"
+    ],
+    "setupFilesAfterEnv": [
+      "./jest/setupTests.js"
+    ]
+  },
```
Comment (Patys) [inline package.json:58]: config to separate file

File: package.json [hunk 2]
```diff
@@ -38,13 +39,26 @@
...
   "husky": {
     "hooks": {
+      "pre-push": "yarn test && yarn lint",
```
Comment (Patys) [inline package.json:61]:
```suggestion
      "pre-push": "yarn prepush",
```
please add separate command

File: src/tests/StickyParallaxHeader.test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react-native';
+import StickyParallaxHeader from '../StickyParallaxHeader';
+
+describe('StickyParallaxHeader snspasot', () => {
+  afterEach(cleanup);
+  test('Snapshot for default', () => {
```
Comment (Patys) [inline src/tests/StickyParallaxHeader.test.js:7]:
```suggestion
  test('Snapshot for empty', () => {
```

**Review by Patys — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (pi0trpietruszewski) [reply package.json:61]: done

File: src/tests/StickyParallaxHeader.test.js (hunk 3 above)
Comment (pi0trpietruszewski) [reply src/tests/StickyParallaxHeader.test.js:7]: done

**Review by Patys — APPROVED** [review]

**Review by IdaszakDaniel — APPROVED** [review]

**Review by MateuszMatyska — APPROVED** [review]

**Pull request merged**